### PR TITLE
feat(veeam-backup): add check for organization on onboarding

### PR DESCRIPTION
**Pull request by chipp972**
ref: MANAGER-15310

| Question         | Answer
| ---------------- | ---
| Branch?          | `develop` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-15310 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

### CODE REVIEW

File: packages/manager/apps/veeam-backup/src/pages/order-veeam/OrderVeeamStep2.component.tsx [hunk 1]
```diff
@@ -96,11 +96,8 @@ export const OrderVeeamStep2: React.FC = () => {
     <>
       <Title className="block mb-9">{t('choose_org_title')}</Title>
       {isLoading && <Loading className="mb-5" />}
-      {!isLoading && data?.pages[0].data.length === 0 && (
-        <OsdsMessage type={ODS_MESSAGE_TYPE.warning}>
-          {t('no_organization_message')}
-          <VCDOrgInfoLink label={t('no_organization_link')} />
-        </OsdsMessage>
+      {!isLoading && (
+        <NoOrganizationMessage organizationList={data?.pages[0].data} />
```
Comment (tibs245) [inline packages/manager/apps/veeam-backup/src/pages/order-veeam/OrderVeeamStep2.component.tsx:100]: If you have an error. You have not JS error ?

I don't known `data?.pages[0]` is defined

File: packages/manager/apps/veeam-backup/src/pages/onboarding/Onboarding.page.tsx [hunk 2]
```diff
@@ -35,18 +43,32 @@ export default function Onboarding() {
...
-    </OnboardingLayout>
+    <>
+      {isError && (
+        <OsdsMessage className="mb-9" type={ODS_MESSAGE_TYPE.error}>
+          {error.message}
+        </OsdsMessage>
+      )}
+      {!isLoading && (
+        <NoOrganizationMessage organizationList={organizationList} />
+      )}
```
Comment (tibs245) [inline packages/manager/apps/veeam-backup/src/pages/onboarding/Onboarding.page.tsx:54]: We show this message if `isError` ?

**Review by tibs245 — APPROVED** [review]

**Review by tibs245 — COMMENTED** [review]
Test unit no pass 

 ✖ Then User sees 1 guide # e2e/step-definitions/onboarding.step.ts:26

File: packages/manager/apps/veeam-backup/src/pages/onboarding/Onboarding.page.tsx (hunk 2 above)
Comment (chipp972) [reply packages/manager/apps/veeam-backup/src/pages/onboarding/Onboarding.page.tsx]: No only if there is no organization

File: packages/manager/apps/veeam-backup/src/pages/order-veeam/OrderVeeamStep2.component.tsx (hunk 1 above)
Comment (chipp972) [reply packages/manager/apps/veeam-backup/src/pages/order-veeam/OrderVeeamStep2.component.tsx]: data is undefined if there is an error but I added a check for a isError
Thanks

**Review by tibs245 — APPROVED** [review]

**Review by darsene — APPROVED** [review]

**Pull request merged**